src/pages/Homepage/Homepage.tsx

```diff
@@ -89,6 +95,7 @@ const Homepage = () => {
           filter={{
             name: "Top Headlines",
```

Comment:
we can use the enum here?

Reply:
No, the enum looks like this: 
export enum ENDPOINTS {
  topheadlines = "top-headlines",
  everything = "everything",
}

Reply:
ok